hydrolib/core/io/ini/util.py

```diff
@@ -58,6 +58,40 @@ def split(v: Any):
     return validator(*field_name, allow_reuse=True, pre=True)(split)
 
 
+def make_list_length_root_validator(
+    *field_names, length_name: str, length_incr: int = 0, id_name: str
```

Comment:
It is good practice to put the optional argument at the end of the method signature. 
In this case, you wouldn't need to specify the key `id_name` each time you call the method.

Reply:
I will change it, but in this case: aren't keywords arguments needed anyway in the call? Since they form the separation between the variadic positional arguments and the keywords arguments, right?

Reply:
Yes that is true actually!